tests/test.cpp

```diff
@@ -157,8 +158,42 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
-  TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  {
+    flatbuffers::Verifier verifier(flatbuf, length);
+    TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  }
+
+  std::vector<uint8_t> test_buff;
+  test_buff.resize(length * 4 + 4);
+  std::memcpy(&test_buff[0], flatbuf , length);
+  std::memcpy(&test_buff[length], flatbuf , length);
+  std::memcpy(&test_buff[2 * length + 1], flatbuf , length);
```

Comment:
Removed as requested. 
Unaligned (**starting address**) seems to work well now, I hope nobody will break it in the future, because there is no specification in the docs that they must be aligned, and people might keep them into an unaligned address.